_posts/2019-12-12-pulp-3-is-GA.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Pulp 3.0 is Generally Available
+author: Brian Bouterse
+tags:
+  - 3.0
+---
+The Pulp developer community is excited to announce the general availability of the following:
+
+* [pulp_rpm 3.0](https://pulp-rpm.readthedocs.io/en/3.0/) to manage RPM content
+* [pulp_container 1.0](https://pulp-container.readthedocs.io/en/1.0/) to manage Docker content
+* [pulp_file 0.1](https://pulp-file.readthedocs.io/en/0.1.0/) to manage Files
+
+These plugins are compatible with the [pulpcore 3.0.0](https://docs.pulpproject.org/) release which
+is also Generally Available today!
+
+### RPM Features
+
+pulp_rpm can sync, organize, and distribute the following content types currently:
+
+* RPMs
+* SRPMs
+* Advisory Content, i.e. Errata
+* Distribution Trees (kickstart trees)
+* Yum Metadata, e.g. comps.xml
+* moduleMD content
+
+You can upload the following to pulp_rpm:
+
+* RPM content
+* SRPM
+* Advisory
+
+### Container Features
+
+pulp_container can sync, organize, upload and distribute Docker content
+
+### File Content
+
+pulp_file can sync, organize, upload, and distribute File content.
+
+
+## Installation
+
+See the installation docs (links below) for each plugin on how to install. They all use the same
+installer so they are very similar. Plugins are capable of being installed all together on one
+installation.
+
+* [pulp_rpm installation]() to manage RPM content
```

Comment:
Missing link, probably this one: https://pulp-rpm.readthedocs.io/en/latest/installation.html

Reply:
Fixed, ty